basic/check_success/boolop.cvc

```diff
@@ -11,8 +11,8 @@ void f()
   float g = 3.14 * 2.0;
   bool h = true * false;
 
-  int i = 4/3;
-  int j = 6/0;
+  float i = 4/3;
```

Comment:
Doesn't integer division produce an integer? Changing this to float should result in a type error